classes/security.php

```diff
@@ -176,7 +176,7 @@ public static function xss_clean($value, array $options = array())
 				import('htmlawed/htmlawed', 'vendor');
 			}
 
-			return htmLawed($value, array_merge(array('safe' => 1, 'balanced' => 0), $options));
+			return htmLawed($value, array_merge(array('safe' => 1, 'balanced' => 0), $options), spec);
```

Comment:
Shouldn't this be `$spec` instead of `spec`?

Reply:
OFC! Sorry! Keep up the good work!